draft-ietf-capport-architecture.xml

```diff
@@ -496,6 +496,189 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
+               is that whether the identifiers are good depends heavily on the capabilities
+               of the components and where in the network the components exist.
+           </t>
+           <section anchor="id_example_interface" title="Physical Interface">
+            <t>
+               The physical interface by which the User Equipment is attached to the
+               network can be used to identify the User Equipment. This identifier has
+               the property of being extremely difficult to spoof: the User Equipment is
+               unaware of the property; one User Equipment cannot manipulate its
+               interactions to appear as though it is another.
+           </t>
+
+            <t>
+               Further, if only a single User Equipment is attatched to a given physical
+               interface, then the identifier will be unique. If multiple User Equipment
+               is attached to the network on the same physical interface, then this property
+               is not appropriate.
+           </t>
+
+            <t>
+               Another consideration related to uniqueness of the User Equipment is that
+               if the attached User Equipment changes, both the API server and the
+               Enforcement Device must invalidate their state related to the User Equipment.
+           </t>
+
+            <t>
+
+               The Enforcement Device needs to be aware of the physical interface,
+               which constrains the environment: it must either be part of the device providing
+               physical access (e.g.. implemented in firmware), or packets traversing the network
+               must be extended to include information about the source physical interface (e.g.
+               a tunnel).
+           </t>
+
+            <t>
+               The API server faces a similar problem, implying that it should co-exist with the
+               Enforcement Device, or that the enforcement device should extend requests to it
+               with the identifying information.
+            </t>
+           </section>
+           <section anchor="id_example_IP_address" title="IP Address">
+            <t>
+                A natural identifier to consider is the IP address of the User Equipment.
+                At any given time, no device on the network can have the same IP address
+                without causing the network to malfunction, so it is appropriate from the
+                perpsective of uniqueness.
+            </t>
+
+            <t>
+                However, it may be possible to spoof the IP address, particularly for
+                malicious reasons where proper funtioning of the network is not necessary
+                for the malicious actor. Consequently, any solution using the IP address
+                should proactively try to prevent spoofing of the IP address. Similarily,
```

Comment:
You might add: "Demonstrations of return routeability, such as that required for TCP connection establishment, might be sufficient defense against spoofing, though this might not be sufficient in networks that use broadcast media (such as some wireless networks)."

Reply:
Done